docs/adr/0005-multitenancy-approach-for-marain.md

```diff
@@ -0,0 +1,263 @@
+# Multitenancy approach for Marain
+
+## Status
+
+Proposed
+
+## Context
+
+Tenancy has been made a first class citizen of all Marain services, however this by itself is not enough to make the system truly multitenanted. In order to do this, we need to determine how tenants should be created, managed and used within the Marain "world".
```

Comment:
"has been made" makes that sounds like a change. It has always been a first class citizen.